src/services/kratos/identity.ts

```diff
@@ -63,6 +63,18 @@ export const IdentityRepository = (): IIdentityRepository => {
         hasNext = page !== undefined && totalCount !== 0
       }
 
+      // * fixes bug in kratos res.headers.link not returning last page properly
```

Comment:
Has this been raised upstream? Have you contacted them on slack? Given:
> This value is currently an integer, but it is not sequential. The value is not the page number, but a reference. The next page can be any number and some numbers might return an empty list.

I don't see how this will fix the issue.

Reply:
yep just posted this topic in slack https://ory-community.slack.com/archives/C012RJ2MQ1H/p1673618866406469